client/src/components/ScoreComponent/package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Also please remove `package.json` file since we are now using `index.js` to easily export our components.